### PR TITLE
[ADD] social_media_icon_extention_xing

### PR DESCRIPTION
**Pull request by Benniphx**
Xing Extension for the social media icons from the odoo core

### CODE REVIEW

File: social_media_icon_extention_xing/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
```
Comment (pedrobaeza) [inline social_media_icon_extention_xing/README.md]: Can you please convert it to RST format, as Markdown is not supported by Odoo?

**Comment by pedrobaeza** [comment]
Module name can be simplified to _social_media_xing_

File: social_media_icon_extention_xing/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Currently added:
+
+Xing
+
+</center>
+
+
+Integration example:
```
Comment (pedrobaeza) [inline social_media_icon_extention_xing/README.md]: You can call this section `Usage`, and add another one called `Configuration` to explain where to specify the URL for the link (Settings > Website and so on)

Comment (pedrobaeza) [inline social_media_icon_extention_xing/README.md]: Mention also that by default, icon is shown on the sections where it appears (footer, about us...)

**Comment by pedrobaeza** [comment]
Some remarks, but the rest :+1: (tried on runbot)

File: social_media_icon_extention_xing/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+{
+    'name': "Xing Social Media Icon Extension",
+    'summary': """Xing Extension for the social media icons from the odoo
+    core""",
+    'author': "bloopark systems GmbH & Co. KG, "
+              "Odoo Community Association (OCA)",
+    'website': "http://www.bloopark.de",
+    'license': 'AGPL-3',
+    'category': 'Social Media',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline social_media_icon_extention_xing/__openerp__.py]: Put new version numbering 8.0.1.0.0 (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)

File: social_media_icon_extention_xing/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    'depends': [
+        'base',
+        'website',
+        'website_blog'
```
Comment (LeartS) [inline social_media_icon_extention_xing/__openerp__.py]: You can just depend on `website_blog` as the other two are implied.

Comment (Benniphx) [inline social_media_icon_extention_xing/__openerp__.py]: I changed it

**Comment by Benniphx** [comment]
@pedrobaeza changed all review issues

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**